style.css

```diff
@@ -0,0 +1,228 @@
+/* My body config */
+body {
+  margin: 0;
+  padding: 0;
+  background: white;
+  box-sizing: border-box;
+  font-family: 'Roboto', sans-serif;
+  height: auto;
+}
+
+/* initialize the .header style: sizes, colors and fonts */
+.header {
+  padding-top: 1px;
+  height: 682px;
+  margin: 0;
+  color: white;
+  background-color: black;
+  width: 100%;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+}
```

Comment:
- [ ] Great work styling your headline 💯. However, please kindly make sure that you use the correct background color in order to stick as much as possible to the Figma design, currently, your background color is `black` and the Figma design is `#1c1a19` 🤓.